roles/setup_gitea/templates/service_gitea.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "gitea"
+  namespace: "{{ sg_namespace }}"
+spec:
+  type: "ClusterIP"
+  ports:
+    - name: http
+      port: 3000
+      targetPort: 3000
+      protocol: TCP
+  selector:
+    app: gitea
```

Comment:
Missed line break at the end of the file

Reply:
Done